src/main/java/com/datatorrent/netlet/DefaultEventLoop.java

```diff
@@ -513,14 +512,18 @@ public String toString()
 
             });
           }
-        }
-        catch (IOException ie) {
-          l.handleException(ie, DefaultEventLoop.this);
+        } catch (UnresolvedAddressException e) {
+          logger.error("Inet Address {} is not resolvable.", address, e);
```

Comment:
Will it be OK to log this as the error as we need it for diagnostic in production deployment? We need the exception to be logged and if it is not logged by netlet, it will not be logged or handled anywhere outside of AbstractClient. Another approach will be to change `AbstractClient.handleException()` to log using error level. Netlet consumers may overwrite `handleException()` and log or handle exception the way they need.